setup.py

```diff
@@ -1,35 +1,49 @@
+#!/usr/bin/env python3
 # coding: utf-8
+"""Wavefront Pyformance Library.
 
-"""Wavefront Pyformance Library
-
-        The Wavefront Pyformance library provides Wavefront reporters
-        (via proxy and direct ingestion) and a simple abstraction for
-        tagging at the host level.
-        It also includes support for Wavefront delta counters.
+The Wavefront Pyformance library provides Wavefront reporters
+(via proxy and direct ingestion) and a simple abstraction for
+tagging at the host level.
+It also includes support for Wavefront delta counters.
 """
 
-
+import os
 try:
     import setuptools
 except ImportError:
     import sys
     sys.exit('Failed to import setuptools. Do pip install setuptools first.')
 
+with open(os.path.join(os.path.abspath(os.path.dirname(__file__)),
+                       'README.md')) as fd:
+    LONG_DESCRIPTION = fd.read()
 
 setuptools.setup(
     name='wavefront_pyformance',
```

Comment:
@sushantdewan123 @vikramraman, should we update the project name with the dash '-' instead of underscore '_'?

Reply:
Will the original `wavefront_pyformance` be available if we update it to `-`?

Reply:
Currently, it can be installed using both
`$ pip install wavefront-pyformance` and
`$ pip install wavefront_pyformance`
hope it works the other way too ;-)

Reply:
Also
```
>>> import pkg_resources
>>> pkg_resources.safe_name('wavefront_pyformance')
'wavefront-pyformance'
```

Reply:
I guess let's just use `wavefront-pyformance` to make it consistent. Change the README installation part as well.